pandas/tests/generic/test_finalize.py

```diff
@@ -491,13 +499,18 @@ def test_finalize_called_eval_numexpr():
     ],
     ids=lambda x: f"({type(x[0]).__name__},{type(x[1]).__name__})",
 )
+@pytest.mark.filterwarnings("ignore:Series.attrs is deprecated:FutureWarning")
 def test_binops(request, args, annotate, all_binary_operators):
     # This generates 624 tests... Is that needed?
     left, right = args
+
+    warn_msg = "(DataFrame|Series).attrs is deprecated"
     if isinstance(left, (pd.DataFrame, pd.Series)):
-        left.attrs = {}
+        with tm.assert_produces_warning(FutureWarning, match=warn_msg):
```

Comment:
can probably remove this if you're already filtering the warnings from the test?